src/battle-tooltips.ts

```diff
@@ -118,10 +118,13 @@ class ModifiableValue {
 		this.maxValue = maxValue;
 		return true;
 	}
+	round(value: number) {
+		return value ? Number(value.toFixed(2)) : 0;
+	}
 	toString() {
 		let valueString;
-		const roundedValue = this.value ? Number(this.value.toFixed(2)) : 0;
-		const roundedMaxValue = this.maxValue ? Number(this.maxValue.toFixed(2)) : 0;
+		const roundedValue = this.round(this.value);
+		const roundedMaxValue = this.round(this.maxValue);
```

Comment:
Do we still need these intermediate values? I'd guess no.

Reply:
Oh, yeah, fair, changed.